src/routes/transactions/transactions.service.ts

```diff
@@ -295,13 +313,16 @@ export class TransactionsService {
     paginationData: PaginationData,
   ): Promise<Page<QueuedItem>> {
```

Comment:
This would also benefit similar changes (as well as other functions in this service).

Reply:
[9db8374](https://github.com/safe-global/safe-client-gateway-nest/pull/610/commits/9db8374f600be9d8fbe58d5c91761aef0ea8b5cf)